.github/workflows/scripts/linux/generate-cmake-qt.sh

```diff
@@ -34,5 +34,5 @@ cmake \
 	-DCMAKE_PREFIX_PATH="$HOME/deps" \
 	-DUSE_SYSTEM_SDL2=ON \
 	-DUSE_SYSTEM_ZSTD=OFF \
-	-DDISABLE_ADVANCE_SIMD=TRUE
-
+	-DDISABLE_ADVANCE_SIMD=TRUE \
+	-DUSE_DBUS
```

Comment:
CMake Error: Parse error in command line argument: USE_DBUS
 Should be: VAR:type=value

Reply:
Fixed